src/components/MyPage.js

```diff
@@ -77,6 +58,49 @@ class MyPage extends Component {
     UserUtil.getRequestSetting(this.onRequestCallback.bind(this));
   }
 
+  getProfileImage(status) {
```

Comment:
It is possible not to use 'image';
`return { uri: status.profile_picture };`
and
`return require('../resources/pattern.png');`